scripts/globals/events/egg_hunt_egg-stravaganza.lua

```diff
@@ -0,0 +1,898 @@
+            { 128, -153.700,  -4.000,  -33.000, "0x00006C0500000000000000000000000000000000", },
+            { 224, -362.004, -10.002, -173.953, "0x00006C0500000000000000000000000000000000", },
+            { 224, -353.928, -10.002, -182.068, "0x00006C0500000000000000000000000000000000", },
+        },
+        helpers =
+        {
+            { 224, -353.000, -10.000, -181.080, "0x01001E0582117120173017401750006000700000", },
+            { 224, -360.900, -10.000, -173.000, "0x01001E0682117120173017401750006000700000", },
+            { 127, -155.000,  -4.000,   33.000, "0x01001E0782117120173017401750006000700000", },
+            { 127, -155.000,  -4.000,   27.000, "0x01001E0882117120173017401750006000700000", },
+        },
+    },
+
+    [xi.zone.WINDURST_WATERS] =
+    {
+        cs = 969,
+        decorations =
+        {
+            { 192, 164.500, -0.250, -30.000, "0x00006D0500000000000000000000000000000000", },
+            { 192, 155.500, -0.250, -30.000, "0x00006D0500000000000000000000000000000000", },
+            {  64, -46.800, -4.916, 227.570, "0x00006D0500000000000000000000000000000000", },
+            {  64, -33.200, -4.982, 227.570, "0x00006D0500000000000000000000000000000000", },
+        },
+        helpers =
+        {
+            { 192, 165.000, -0.250, -28.800, "0x01001E0582117020173017401750006000700000", },
+            { 192, 164.000, -0.250, -28.800, "0x01001E0682117020173017401750006000700000", },
+            { 192, 155.500, -0.250, -28.800, "0x01001E0782117020173017401750006000700000", },
+            {  63, -46.800, -4.900, 226.220, "0x01001E0882117020173017401750006000700000", },
+            {  63, -33.200, -4.900, 226.220, "0x01001E0182117020173017401750006000700000", },
+        },
+    },
+
+    [xi.zone.WINDURST_WOODS] =
+    {
+        cs = 785,
+        decorations =
+        {
+            { 128,  107.630, -5.000, -33.200, "0x00006D0500000000000000000000000000000000", },
+            { 128,  107.630, -5.000, -46.800, "0x00006D0500000000000000000000000000000000", },
+            {   0, -107.000, -5.250,  35.500, "0x00006D0500000000000000000000000000000000", },
+            {   0, -107.000, -5.250,  44.500, "0x00006D0500000000000000000000000000000000", },
+        },
+        helpers =
+        {
+            {   0, -105.500, -5.250,  35.500, "0x01001E0282117020173017401750006000700000", },
+            {   0, -105.500, -5.250,  44.500, "0x01001E0782117020173017401750006000700000", },
+            { 127,  106.500, -5.000, -33.200, "0x01001E0382117020173017401750006000700000", },
+            { 127,  106.500, -5.000, -46.800, "0x01001E0482117020173017401750006000700000", },
+            { 127, -155.000, -4.000, -27.000, "0x01001E0100107120173017401750006000700000", },
+            { 127, -155.000, -4.000, -33.000, "0x01001E0282117120173017401750006000700000", },
+        },
+    },
+}
+
+local messageOffset =
```

Comment:
I like this pattern for offsets a lot.  It makes things much easier to look up when we're using them